spec/listformat.html

```diff
@@ -91,7 +91,7 @@ <h1>Internal slots</h1>
       </p>
 
       <ul>
-        <li>[[LocaleData]].[[&lt;_locale_&gt;]] is a Record which has three fields [[conjunction]], [[disjunction]], and [[unit]]. Each of these is a Record which must have fields with the names of three formatting styles: [[long]], [[short]], and [[narrow]].</li>
+        <li>[[LocaleData]].[[&lt;_locale_&gt;]] is a Record which has three fields [[Conjunction]], [[Disjunction]], and [[Unit]]. Each of these is a Record which must have fields with the names of three formatting styles: [[Long]], [[Short]], and [[Narrow]].</li>
```

Comment:
These also cannot change while the [Intl.ListFormat constructor](https://tc39.es/ecma402/#sec-Intl.ListFormat) uses lowercase option values as lookup keys.
```suggestion
        <li>[[LocaleData]].[[&lt;_locale_&gt;]] is a Record which has three fields [[conjunction]], [[disjunction]], and [[unit]]. Each of these is a Record which must have fields with the names of three formatting styles: [[long]], [[short]], and [[narrow]].</li>
```